README.md

```diff
@@ -142,6 +142,21 @@ status = "generate"
     version = "3.18"
 ```
 
+For global functions, the members can be configured by configuring the `Gtk.*` object:
+
+```toml
+[[object]]
+name = "Gtk.*"
+status = "generate"
+    [[object.member]]
+    name = "stock_list_ids"
+    # allows to ignore global functions
+    ignore = true
+```
+
+Note that you must not place `Gtk.*` into the `generate` array and
+additionally configure its members.
```

Comment:
It does, but I think it's worth repeating here because "Gtk.*" is a bit special. Conceptually you would think of the functions as the things to generate, not "Gtk.*". I can remove it if you prefer